CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
no need

Reply:
It is need though if we open-source a codebase, but I can remove it on your choice ?
What do u say sir ?

Reply:
remove that file only